### PR TITLE
Message managers feature

### LINKED ISSUE #3146: Add @managers handle on comments field
Add a `@managers` handle on tasks and projects chat to allow users to message the project author, the organization managers and the Project Manager team members.

### PR DESCRIPTION
**Pull request by JorgeMartinezG**
Closes #3146. The trigger was modified to use # instead of @

### CODE REVIEW

File: backend/services/messaging/message_service.py [hunk 1]
```diff
@@ -450,7 +451,44 @@ def resend_email_validation(user_id: int):
         SMTPService.send_verification_email(user.email_address, user.username)
 
     @staticmethod
-    def _parse_message_for_username(message: str) -> List[str]:
+    def _get_managers(message: str, project_id: int) -> List[str]:
+        parser = re.compile(r"((?<=#)\w+|\[.+?\])")
+        parsed = parser.findall(message)
+
+        prj = None
```
Comment (willemarcel) [inline backend/services/messaging/message_service.py:458]: let's rename it to project

File: backend/services/messaging/message_service.py [hunk 2]
```diff
@@ -450,7 +451,44 @@ def resend_email_validation(user_id: int):
...
+        if "author" in parsed or "managers" in parsed:
+            prj = Project.query.get(project_id)
+
+        if prj is None:
+            return []
+
+        prj_usernames = [prj.author.username]
```
Comment (willemarcel) [inline backend/services/messaging/message_service.py:465]: let's rename to project_managers

File: backend/services/messaging/message_service.py [hunk 3]
```diff
@@ -450,7 +451,44 @@ def resend_email_validation(user_id: int):
...
+
+        if "managers" not in parsed:
+            return prj_usernames
+
+        teams = [t for t in prj.teams if t.role == TeamRoles.PROJECT_MANAGER.value]
+        team_members = [
+            [
+                u.member.username
+                for u in t.team.members
+                if u.function == TeamMemberFunctions.MANAGER.value
```
Comment (willemarcel) [inline backend/services/messaging/message_service.py:475]: On the case of Project Manager teams, we should get all active users, not only the team managers.

**Comment by JorgeMartinezG** [comment]
@willemarcel applied feedback

**Pull request merged**